languagetool-core/pom.xml

```diff
@@ -208,8 +208,8 @@
             <artifactId>annotations</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.jetbrains.intellij.deps</groupId>
-            <artifactId>trove4j</artifactId>
+            <groupId>it.unimi.dsi</groupId>
+            <artifactId>fastutil</artifactId>
```

Comment:
Maybe use a stripped version from IJ?

Reply:
Oh, yes, I kind of forgot about it. But we decided to just use the official `fastutil-core`, as the IJ version does not worth the trouble to use outside of IJ.

Reply:
I hope it's OK to add a 22 MB dependency

Reply:
`fastutil-core` is only `6.1MB`. See the commit below.